rich/__main__.py

```diff
@@ -97,7 +97,7 @@ def comparison(renderable1: RenderableType, renderable2: RenderableType) -> Tabl
 
     table.add_row(
         "Asian\nlanguage\nsupport",
-        ":flag_for_china:  该库支持中文，日文和韩文文本！\n:flag_for_japan:  ライブラリは中国語、日本語、韓国語のテキストをサポートしています\n:flag_for_south_korea:  이 라이브러리는 중국어, 일본어 및 한국어 텍스트를 지원합니다",
+        ":flag_for_china:  该库支持中文，日文和韩文文本！\n:flag_for_japan:  ライブラリは中国語、日本語、韓国語のテキストをサポートしています\n:flag_for_south_korea:  이 라이브러리는 중국어, 일본어 및 한국어 텍스트를 지원합니다\n:flag_for_taiwan:  本函式庫支援中日韓文字元！",
```

Comment:
Sorry, this will break the tests which use this content. And it will require the screenshots to be generated. These were just examples and not meant to be exhuastive.

Reply:
That's sad, but I understand this change requires extra work. It's removed.